optimade/models/optimade_json.py

```diff
@@ -244,6 +244,11 @@ class Implementation(BaseModel):
         description="A dictionary providing details about the maintainer of the implementation.",
     )
 
+    issue_tracker: Optional[Union[AnyUrl, jsonapi.Link]] = StrictField(
```

Comment:
Ah we didn't include this yet 😮

Reply:
Yeah, the versioning was always the issue, and without the spec releasing we're left in no-mans-land. We'll just have to push it through as we did with the mass changes

Reply:
So we should just release v0.16 straight after this and make the PR to the spec repo